nixos/modules/misc/nixpkgs.nix

```diff
@@ -130,26 +131,35 @@ in
       '';
     };
 
-    crossSystem = mkOption {
-      type = types.nullOr types.attrs;
-      default = null;
+    localSystem = mkOption {
+      type = types.attrs; # TODO utilize lib.systems.parsedPlatform
+      example = lib.systems.examples.aarch64-multiplatform;
```

Comment:
This would be a fully elaborated platform, and quite large. I could put something minimal too.